<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setting</class>
 <widget class="QDialog" name="Setting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <widget class="QWidget" name="themePage">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>30</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>主题</string>
     </property>
    </widget>
    <widget class="QComboBox" name="themeChoose">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>10</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolButton" name="themeBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Theme</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
